service/topology_coordinator.cc

```diff
@@ -825,8 +826,10 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
             guard = co_await start_operation();
         }
         topology_mutation_builder builder(guard.write_timestamp());
-        builder.set_fence_version(_topo_sm._topology.version);
-        auto reason = ::format("advance fence version to {}", _topo_sm._topology.version);
+        // Other nodes are guaranteed to be drained on success only up to the version which was current
+        // prior to barrier_and_drain RPCs were sent.
+        builder.set_fence_version(version);
+        auto reason = ::format("advance fence version to {}", version);
```

Comment:
@gleb-cloudius @kbr-scylla Please review.

Reply:
How do I even find out on the github what patch this change belongs to? All I can see from your ping is a very small context without any explanation.

Reply:
@gleb-cloudius it's fad6c41cee8916df0d20e37ec36a9cc9b9012e89

Reply:
@tchaikov Thanks, but how do you find it through the github GUI?

Reply:
![image](https://github.com/scylladb/scylladb/assets/59071/b3b898eb-4d02-487e-8455-be6ff46b4d9c)

click the top-left link of "[service/topology_coordinator.cc](https://github.com/scylladb/scylladb/pull/18026/files/fad6c41cee8916df0d20e37ec36a9cc9b9012e89#diff-7367612c80d45bdd40bc28c7b3edc3a163a53cb802b839c3eae1112eb0611f28)", et voila!

Reply:
> click the top-left link of "[service/topology_coordinator.cc](https://github.com/scylladb/scylladb/pull/18026/filesfad6c41cee8916df0d20e37ec36a9cc9b9012e89#diff-7367612c80d45bdd40bc28c7b3edc3a163a53cb802b839c3eae1112eb0611f28)", et voila!

That what I did. It does not take me to the commit with the commit message, only a slightly larger context.

Reply:
oic. the link was 

`https://github.com/scylladb/scylladb/pull/18026/files/fad6c41cee8916df0d20e37ec36a9cc9b9012e89`, but you need to change it to 

`https://github.com/scylladb/scylladb/pull/18026/commits/fad6c41cee8916df0d20e37ec36a9cc9b9012e89 ` 

by changing "files" to "commits". sorry for missing it.

Reply:
Thanks you. So I need to change the link manually? (yet another confirmation how bad github PRs are :)).

Reply:
sure thing! yeah, guess i am already used to it..